doorbell/doorbell_backend/src/detection.py

```diff
@@ -1,23 +1,22 @@
+# TODO: prune imports to specific functions from libraries to reduce .exe size
 import cv2
 import sys
-import socket  # maybe use this for sending files, but if there's a library that does HTTPS connections use that instead
 from pathlib import Path
 import time
 from datetime import datetime
```

Comment:
Do we need multiple lines for the imports here ? Can we do one `from` block for all imports from `datetime`

Reply:
Import pruning is one of the TODO's, and should be done to reduce the footprint. The requirements.txt will also be filled out to allow for easier pip installations of required libraries.